src/measure_theory/haar_measure.lean

```diff
@@ -285,18 +285,24 @@ begin
   have h2V₀ : (1 : G) ∈ V₀, { simp only [mem_Inter], rintro ⟨V, hV⟩ h2V, exact hV.2 },
   refine ⟨prehaar K₀.1 V₀, _⟩,
   split,
-  { apply prehaar_mem_haar_product K₀, use 1, rwa h1V₀.interior_eq  },
+  { apply prehaar_mem_haar_product K₀, use 1, rwa h1V₀.interior_eq },
   { simp only [mem_Inter], rintro ⟨V, hV⟩ h2V, apply subset_closure,
     apply mem_image_of_mem, rw [mem_set_of_eq],
     exact ⟨subset.trans (Inter_subset _ ⟨V, hV⟩) (Inter_subset _ h2V), h1V₀, h2V₀⟩ },
 end
 
 /-!
-### The Haar measure on compact sets
+### chaar
```

Comment:
Is this what you want?

Reply:
```suggestion
### Lemmas about `chaar`
```

Reply:
The previous section title was incorrect (or at least misleading), so I did want to change it. I made it better now.